tests/integration/models/keras/test_predictions.py

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pytest
 import numpy as np
 import tensorflow as tf
+from sklearn.preprocessing import StandardScaler
```

Comment:
I think this one assumes Gaussian distribution of the variables? min-max normalization would probably work better?